docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -19,7 +19,7 @@ The minimal required version of .NET Framework is 4.7.2.
     - Ensure C#, VB, MSBuild, .NET Core and Visual Studio Extensibility are included in the selected work loads
     - Ensure "Use previews of the .NET Core SDK" is checked in Tools -> Options -> Environment -> Preview Features
     - Restart Visual Studio
-1. [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/6.0) [Windows x64 installer](https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-6.0.100-windows-x64-installer)
+1. [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/6.0) [Windows x64 installer](https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-6.0.200-windows-x64-installer)
```

Comment:
Do we need to be more specific? Currently it should be 6.0.2xx since we are rolling forward based on `latestPatch`, right?

```suggestion
1. [.NET 6.0.2xx SDK](https://dotnet.microsoft.com/download/dotnet-core/6.0) [Windows x64 installer](https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-6.0.200-windows-x64-installer)
```

Reply:
> since we are rolling forward based on `latestPatch`

Does this affect what SDK gets installed automatically when running a CI build?

Reply:
No, the dotnet install script will install the version specified in the global.json.

Reply:
We could also just be specific and say "6.0.200" but thought it might be good to provide additional clarity.

Reply:
I think you're right. It might even be better to delete the windows installer link because it's harder to keep up to date. Just make people obtain the latest patch for 2xx, for example.

Reply:
For now I'll just take what you've suggested in order to get the change in.